specifyweb/frontend/js_src/lib/components/FormSliders/AttachmentsCollection.tsx

```diff
@@ -23,13 +25,14 @@ export function AttachmentsCollection({
 
   const [scale = defaultScale] = useCachedState('attachments', 'scale');
 
-  const attachments = React.useMemo(
-    () =>
-      collection.models.map(
-        (resource) =>
-          serializeResource(resource) as SerializedResource<Attachment>
-      ),
-    [collection]
+  const attachments: RA<SerializedResource<Attachment>> = filterArray(
+    Array.from(collection.models, (model) => {
+      if (model.specifyModel.name.includes('Attachment')) {
+        const record = serializeResource(model) as SerializedResource<CollectionObjectAttachment>
+        return serializeResource(record.attachment) as SerializedResource<Attachment>
+      }
+      return undefined;
+    })
```

Comment:
huh, why was this changed again to a more complex version?
does the simpler version not work?

Reply:
no it didn't... we have found this solution with Jason

Reply:
Got it. thanks for clarifying